logging/derivation/src/main/scala/tofu/logging/derivation/loggingMid.scala

```diff
@@ -0,0 +1,31 @@
+package tofu.logging
+package derivation
+
+import tofu.higherKind.derived.HigherKindedMacros
+import derevo.DerivationKN3
+import derevo.DerivationKN11
+import tofu.logging.bi.LoggingBiMid
+import tofu.logging.bi.LoggingBiMidBuilder
+import derevo.PassTypeArgs
+import derevo.ParamRequire
+
+object loggingMid
```

Comment:
We definitely need some scaladoc here, probably with the link to the docs.
Those are things that have to be described:
- differences between all of `loggingMid*`
- how each method will be logged
- how to configure logging
- where the `Logging` instance will be looked for